lib/fhir_client/client.rb

```diff
@@ -134,8 +134,8 @@ def get_oauth2_metadata_from_conformance
       authorize_extension = 'authorize'
       token_extension = 'token'
       begin
-        conformance = conformance_statement
-        conformance.rest.each do |rest|
+        capabilities = capability_statement
```

Comment:
I'm not sure this variable is used anywhere, other than the line below. (Probably a bit nit-picky: but we probably could drop the temp variable.)